### ISSUE TITLE
New Geocoder Contribution

### ISSUE THREAD

**Issue by dreamorosi** [geopy/geopy#533]
Hello,

I have read the contributing guidelines and saw that PRs that add new geocoders are welcome.

Before starting to work on the new geocoder I wanted to understand if that's still the case. I saw that there are some PR adding geocoders that have been open for a while so I wanted to make sure the repo is still maintained.

After that, assuming that the PR is reviewed, accepted, and merged I wanted also to understand how it would fit into the release schedule of the package. I saw that there are some milestone open but they have no deadline, also the latest release in Pypi is from over a year ago.

In short, I'd like to hear from one of the maintainers if a new geocoder would be welcome and if so, when would be the next planned release to have it published.

Thanks!

**Comment by KostyaEsmukov** [comment]
Hello,

Unfortunately I'm the only active maintainer of this project at the moment, and I don't have much time and power I can dedicate to it, hence the irregularity in releases and PR reviews.

The reason for a lot of open PRs with new geocoders is that most of them are not in the ready to be merged shape, some would require multiple iterations and a lot of effort to get ready, something I don't currently have spare time for.

That said, a good PR with a new geocoder requiring as little effort as possible has very good chances to be merged and released quickly. A "good PR" is a one which uses the same approaches and codestyle as all the other geopy geocoders, which has docs, decent tests, and which describes how I can easily obtain the API credentials required for the tests.

As for the release schedule, there isn't any.
